client/blocks/jitm/templates/modal.jsx

```diff
@@ -53,12 +90,7 @@ export default function ModalTemplate( {
 									<p className="modal__disclaimer">{ line }</p>
 								) ) }
 							</div>
-							<div className="modal__sidebar">
-								<img
-									src={ Visual }
-									alt="list of available plans — premium, business, and ecommerce plans"
-								/>
-							</div>
+							<div className="modal__sidebar">{ modalImage }</div>
```

Comment:
This variable doesn't look necessary, we could inline everything:

```suggestion
							<div className="modal__sidebar">
								<img className={ getClassName() } src={ getModalImage() } alt={ getModalAltText() } />
							</div>
```